.github/workflows/CI.yml

```
@@ -39,3 +39,16 @@ jobs:
           COMMIT_SHA: ${{ github.sha }}
         run: |
           while IFS= read -r line; do echo; sudo -E bash ./main.sh -vt "${line}" || exit 1; echo; done < <(git ls-files '.scripts/*.sh' | sed -E 's/^\.scripts\/(\w+)\.sh$/\1/')
+
+  validate-images:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.3.2
+      - name: Validate Images
+        working-directory: compose
+        env:
+          CI: true
+          COMMIT_SHA: ${{ github.sha }}
+        run: |
+          dirname -- $(grep -Rl 'deprecated: "true"' .) | xargs rm -rf
```

```suggestion
          dirname -- $(grep -Rl 'deprecated: "true"') | xargs rm -rf
```
The `.` isn't needed if you're in the directory.